verification/curator-service/ui/src/components/SourceTable.tsx

```diff
@@ -294,6 +298,7 @@ class SourceTable extends React.Component<Props, SourceTableState> {
                                     />
                                 ),
                             },
+                            { title: 'Format', field: 'format' },
```

Comment:
How about a dropdown with `CSV` and `JSON`?

Reply:
Good idea, let me send you a separate PR for that now.